apiv2/support/enums.py

```diff
@@ -11,21 +11,21 @@
 
 
 @strawberry.enum
-class annotation_file_source_enum(enum.Enum):
+class annotation_file_source_enum(enum.StrEnum):
     dataset_author = "dataset_author"
     community = "community"
     portal_standard = "portal_standard"
 
 
 @strawberry.enum
-class annotation_method_type_enum(enum.Enum):
+class annotation_method_type_enum(enum.StrEnum):
```

Comment:
nit: iirc believe we are adding an additional type here `simulated`.

Reply:
Also added this as a note on the other PR!